docs/guide.md

```diff
@@ -310,6 +310,61 @@ isCurrentUser(uid) { auth != null && auth.uid == uid }
 }
 ```
 
+## Imports
+
+Local & Global type imports are supported.
+
+### Global Module
+
+    import * from {'module'}
+
+Each modules is assumed to have an entry point under the 'node_modules/firebase-bolt-module/index.bolt' filename and will be processed from here.
```

Comment:
What if they are installed under global node modules?

Reply:
On global modules I'm not a fan of using them so preference would be to just not support it. It's a bad practice as you lose track of versioning which is what the package manager is designed to do. Some things like the cli might make sense to be global but actual project code should be self contained so that the build chain is stable.

Even with firebase if we have someone who has a v2 & v3 app, which version of the cli am i running to publish and work with the app? if it's installed in the global module space I would have to upgrade everything or nothing which isn't practical.

Reply:
I agree it's not great to support, we should at least call out that you can't install them globally.